src/Catty/DataModel/VariablesContainer/VariablesContainer.m

```diff
@@ -459,6 +459,26 @@ - (BOOL)isProjectVariableOrList:(UserVariable*)userVarOrList
     return NO;
 }
 
+- (BOOL)isProjectVariable: (UserVariable*)userVariable
+{
+    for (UserVariable *userVariableToCompare in self.programVariableList) {
+        if ([userVariableToCompare.name isEqualToString:userVariable.name]) {
+            return YES;
+        }
+    }
+    return NO;
+}
+
+- (BOOL)isProjectList: (UserList*)userList
+{
+    for (UserList *userListToCompare in self.programListOfLists) {
```

Comment:
Please change to `UserVariable` since the elements of this list are still of type `UserVariable`.